packages/aws-cdk-lib/aws-lakeformation/README.md

```diff
@@ -25,3 +25,87 @@ For more information on the resources and properties available for this service,
 (Read the [CDK Contributing Guide](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md) and submit an RFC if you are interested in contributing to this construct library.)
 
 <!--END CFNONLY DISCLAIMER-->
+
+### Example
+
+Using the lakeformation L1s is not always the most straight forward. Here is an example of creating a glue table and putting lakeformation tags on it. Note: this example uses deprecated constructs and overly permissive IAM roles. This example is meant to give a general idea of using the L1s; it is not production level.
```

Comment:
Nit: I would remove the first sentence and just start with "Here is an example of creating a glue table and ...". Also should we give an example that isn't using the deprecated glue `Table`?